src/main/webapp/components/navbar.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <nav class="blue sticky nav-bar">
+    <a href="index.html" class="bold" id="title">starfish</a>
+    <img src="../assets/starfish.png" id="logo">
+    <div id="action-items">
+      <a href="#" id="post-action">PostANote</a>
+      <div class="dropdown">
+        <button class="dropdown-btn" @click.stop="toggleDropdown">
+          <img id="user-profile" src="../assets/user.svg">
+        </button>
+        <div class="light-gray dropdown-content" v-if="showDropdown" v-click-outside="hideDropdown">
+          <img src="../assets/user.svg" height="115px"/>
+          <p class="bold" id="dropdown-name">{{ user.name }}</p>
+          <p id="dropdown-points">{{ user.points }} points</p>
+          <a class="dropdown-link" href="#">My Profile</a> 
```

Comment:
I think it's useful to add TODO comments when there are things that will need to be added so you can get into the habit of searching your code for TODOs and ensuring cleanup